opm/parser/eclipse/EclipseState/Schedule/Schedule.hpp

```diff
@@ -69,6 +78,8 @@ namespace Opm
         Runspec m_runspec;
         RSTConfig rst_config;
         std::optional<int> output_interval;
+        double sumthin{-1.0};
```

Comment:
I assume this `-1` is some special *I-am-not-in-use/initialized* - can we use `std::optional<double>` instead?